<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoleGeneratorDialog</class>
 <widget class="QDialog" name="RoleGeneratorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Role Generator Config</string>
  </property>
  <layout class="QFormLayout" name="formLayout_5">
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="mBtnBoxOkCancel">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Postgres service configuration</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>PG Config setup</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="pgserviceComboBox"/>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="GrantRolesOnDBCheckBox">
        <property name="text">
         <string>Create Grants on database</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="CreateRolesCheckBox">
        <property name="text">
         <string>Create roles</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="RevokeRolesOnDBCheckBox">
        <property name="text">
         <string>Revoke Grants on database</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="addDbSpecRolesCheckBox">
     <property name="text">
      <string>Use database specific roles instead of cluster specific roles</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mBtnBoxOkCancel</sender>
   <signal>accepted()</signal>
   <receiver>RoleGeneratorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mBtnBoxOkCancel</sender>
   <signal>rejected()</signal>
   <receiver>RoleGeneratorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
